lib/internal/process/policy.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {
+  ERR_MANIFEST_TDZ,
+  ERR_MANIFEST_PARSE,
+} = require('internal/errors').codes;
+const fs = require('fs');
+const { pathToFileURL } = require('internal/url');
+const { URL } = require('url');
+const { Manifest } = require('internal/policy/manifest');
+let manifest;
+module.exports = Object.freeze({
+  __proto__: null,
+  setup() {
+    const manifestSpecifier = process.binding('config').experimentalPolicy;
+    if (typeof manifestSpecifier !== 'string') {
+      manifest = null;
+      return;
+    }
+    const cwdURL = pathToFileURL(process.cwd());
+    cwdURL.pathname += '/';
+    // URL here as it is slightly different parsing
+    // no bare specifiers for now
+    const manifestURL = new URL(manifestSpecifier, cwdURL);
+    try {
+      const json = JSON.parse(fs.readFileSync(manifestURL, 'utf8'), (_, o) => {
+        if (o && typeof o === 'object') {
+          Reflect.setPrototypeOf(o, null);
+          Object.freeze(o);
+        }
+        return o;
+      });
+      manifest = new Manifest(json);
+    } catch (e) {
```

Comment:
I think we should not lose this error.

Reply:
do we have a recommended practice for nesting errors, we don't to my knowledge

Reply:
We don't, but I think a user would like to know where the JSON is malformed, reading the file errors or what else could happen.

Reply:
i can add the error.message but don't know where i should put the error.code, adding both would be a bit confusing.

Reply:
How about not wrapping at all?

Reply:
seems fine, removed the wrapper